protobuf/sync.proto

```diff
@@ -1,13 +1,19 @@
 syntax = "proto3";
 
 package mvds;
-option go_package = "protobuf";
+option go_package = "./;protobuf";
+
+message Offer {
+  bytes group_id = 1;
+  bytes message_id = 2;
+}
 
 message Payload {
   repeated bytes acks = 5001;
   repeated bytes offers = 5002;
   repeated bytes requests = 5003;
   repeated Message messages = 5004;
+  repeated Offer new_offers = 5005;
```

Comment:
I was trying to understand MVDS which is based on bramle sync and it looks like the concept of group(which is like an application or community in status case) is already included in the protocol design itself https://code.briarproject.org/briar/briar-spec/blob/master/protocols/BSP.md#11-system-model 

If that is the case, then what is the need for this new_offers field which has separate group id? I maybe missing something here, but want to get clarity on this.

Reply:
Good question, 

so bramble I believe works a bit differently from us, you communicate directly with the peer, and you can only communicate with contacts, in our case, we share messages with any member of the community, which is potentially quite large. 

I had a look at the code, and it seems that bramble will offer message ids, and the filtering is done after the message is received on the other side (group id information is passed only with the message, rather than with the group id).

In our case, we broadcast messages and we use it for communities, so the potential audience is much larger, and for example for a community, there's many different channels, and not all the members will have access to all channels. 

So we would either just OFFER message id, with the potential issue of having to filter after receiving the message (which would effectively be worse in terms of bandwith), or we derive the group id implicitly by say the topic (any OFFER message for topic `x` will belongs to community `x`), but in this way, we would need to duplicate offers for each different channel that has different permission in a community, which likely would also be more bandwidth (depends on the channel).

So as far as I can see, we have 3 options:

1) GroupID is inferred by the channel the Offer message is sent to
2) GroupID is passed only after requesting a message
3) GroupID is specified on the offer (as per above)

Not sure if I can think of anything else, and 3 seems the best for our scenario, but open to other opinions

Reply:
Thanks for clarification and providing more context. 
I was not aware that access to channels is also specific to users(now that i think about it is required). 
If so, the proposed option seems to be the best way to move ahead. 

One more question or rather clarification: 
In status communitie case the sync is not between peers, rather users of the community since these MVDS messages would be broadcasted that could only be understood/decrypted by community members?

Reply:
> In status communitie case the sync is not between peers, rather users of the community since these MVDS messages would be broadcasted that could only be understood/decrypted by community members?
Yes, that's correct it's never among "peers" as in waku peers, it's always e2e